### PR TITLE
Tag BuildExecutable.jl v0.1.2

### PR DESCRIPTION
**Pull request by attobot**
Repository: [dhoegh/BuildExecutable.jl](https://github.com/dhoegh/BuildExecutable.jl)
Release: [v0.1.2](https://github.com/dhoegh/BuildExecutable.jl/releases/tag/v0.1.2)
Travis: [Travis Build Status](https://travis-ci.org/dhoegh/BuildExecutable.jl/builds/196392819)
Diff: [vs v0.1.1](https://github.com/dhoegh/BuildExecutable.jl/compare/f1a2645924fd3e8efc17aae7cd98af5000466233...e7c0f7956b0de6c687b7194d9a4fc9397a9e5edc)
`requires` vs v0.1.1: no changes
cc: @dhoegh

### CODE REVIEW

**Comment by tkelman** [comment]
You need to add Compat, with a minimum version that implements the features you need, to REQUIRE

**Comment by dhoegh** [comment]
Thank you for your comment. I have added `Compat` as requirement.

File: BuildExecutable/versions/0.1.2/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4
+BinDeps
+Compat 0.14
```
Comment (tkelman) [inline BuildExecutable/versions/0.1.2/requires]: unless you're using a feature that didn't exist in versions of Compat prior to 0.14, this is too tight a bound

**Comment by dhoegh** [comment]
Should I delete the version requirement or what is a reasonable version requirement?

**Comment by tkelman** [comment]
What features are you using from Compat?

**Comment by dhoegh** [comment]
Primarily `@static` and the `is_xxxxx` functions.

**Comment by tkelman** [comment]
That was introduced in Compat 0.7.20.

**Comment by dhoegh** [comment]
Thank you, I will update accordingly.

**Comment by MikeInnes** [comment]
Please check that:
- [ ] The package's CI is green for this tag (if applicable).
- [ ] The version bounds are up to date.

Then reply with `+ready` so I can merge.

**Pull request merged**